docs/oscar_references.bib

```diff
@@ -1067,6 +1067,47 @@ @Book{Ful98
   doi           = {10.1007/978-1-4612-1700-8}
 }
 
+@Article{FKRS20,
```

Comment:
It seems that this paper has been published by the AMS (https://www.ams.org/journals/mcom/2021-90-330/S0025-5718-2021-03624-8/home.html). Please cite this version instead of the arxiv one